Hazel/src/Hazel/Debug/Instrumentor.h

```diff
@@ -9,6 +9,36 @@
 
 namespace Hazel {
 
+	template <size_t N>
+	struct Fixed_String {
+		char data[N];
+	};
+
+	template <size_t N>
+	constexpr auto Clean_Expression(const char(&expr)[N]) {
+		Fixed_String<N> result = {};
+
+		int src_idx = 0;
+		int dst_idx = 0;
```

Comment:
string size is a size_t and again some coding convention (not use `_`)
```suggestion
		size_t srcIndex = 0;
		size_t dstIndex = 0;
```

Reply:
Done